code/upaya/config/application.rb

```diff
@@ -48,5 +48,9 @@ class Application < Rails::Application
     # config.active_job.queue_adapter = :resque
 
     # config.middleware.use Rack::Attack unless Figaro.env.pt_mode == 'on'
+
+    # Need this if we don't recompile before cloudfourndry push
+    #config.assets.compile = false
+
```

Comment:
Great! will remove... again learning :) totally new to ruby. you all will be my mentor here :)